### ISSUE TITLE
Trying to run test example... failing with write.table() error?

### ISSUE THREAD

**Issue by roseisgold** [mskilab/JaBbA#70]
Would love to get some help on how to debug this error--finally got it installed and am trying to test it out on the test data with the command: 
jba ${JABBA_PATH}/junctions.vcf ${JABBA_PATH}/coverage.txt

But am running into this error: 

<img width="1172" alt="Screenshot 2022-09-05 at 16 12 16" src="https://user-images.githubusercontent.com/112878116/188468704-e9713be2-0a90-4964-95c7-8ef91673aecd.png">

Have you seen this before / do you have any thoughts on how to work around it?

**Comment by zining01** [comment]
Hi, thanks for reaching out regarding this issue. No, unfortunately, we have never encountered this problem before. Could you please provide the following information:

- the full error log from JaBbA (jabba.out and jabba.err generated from the command below)
`jba [junction file] [coverage file] > jabba.out 2>jabba.err`
- the file `cmd.args` which should be produced in your working directory by JaBbA
- the file `cmd.args.rds` which should also be produced in the working directory

Thanks!

**Comment by roseisgold** [comment]
Hi! Yes of course, I tried to re-install and re-install some dependencies, but am still getting various errors:

[jabba.out.txt](https://github.com/mskilab/JaBbA/files/9535339/jabba.out.txt)
[jabba.err.txt](https://github.com/mskilab/JaBbA/files/9535332/jabba.err.txt)

My cmd.args: --geno FALSE --indel exclude --tfield tier --iterate 0 --rescue.window 1000 --rescue.all TRUE --nudgebalanced TRUE --edgenudge 0.1 --strict FALSE --allin FALSE --field ratio --maxna 0.5 --ploidy NA --purity NA --ppmethod sequenza --cnsignif 1e-05 --slack 100 --linear FALSE --tilim 6000 --epgap 1e-05 --outdir ./ --name tumor --cores 1 --mem 16 --fix.thres -1 --lp TRUE --ism FALSE --filter_loose FALSE --gurobi FALSE --verbose FALSE --help FALSE --junctions /Library/Frameworks/R.framework/Versions/4.2/Resources/library/JaBbA/extdata//junctions.vcf --coverage /Library/Frameworks/R.framework/Versions/4.2/Resources/library/JaBbA/extdata//coverage.txt

**Comment by zining01** [comment]
hi, thanks for the extra info and sorry you are having so many problems with JaBbA. Does JaBbA create any of the following files in your output directory: `jabba.rds`, `jabba.simple.rds`, `karyograph.rds`? and if so could you please attach them?

as something to try, i pushed something that may temporarily fix your issue to this branch: `https://github.com/mskilab/JaBbA/tree/qc` feel free to pull and give that a shot.

thanks for your patience!

**Comment by roseisgold** [comment]
Hi! I do get those output files—is there an email I can send them to? Git won’t let me attach .rds filetypes. I am giving the qc branch a try right now! 

One more question- do you ever see this warning message upon installation?

**Comment by roseisgold** [comment]
<img width="1219" alt="Screenshot 2022-09-12 at 10 28 36" src="https://user-images.githubusercontent.com/112878116/189611931-18ad45fd-be6f-4d45-b8cc-955390618525.png">

**Comment by roseisgold** [comment]
Sorry for the spam--I cloned your gc branch, ran R CMD build and R CMD check--here is the output from the check:
[00check.log](https://github.com/mskilab/JaBbA/files/9546781/00check.log)

Please let me know how I can help!! Thank you so much!!

**Comment by zining01** [comment]
Yeah, I always see that warning. I think the original developers' idea was to provide some 'syntactic sugar' to allow users to easily get the seqinfo corresponding to a gGraph (e.g. using `seqinfo(gg)`) but it now always creates that warning on startup and we never got around to fixing it.

You can send the files to `zining01@gmail.com`. Did the test example work following reinstallation from the `qc` branch or are you still having the same issue?
